lib/screens/about_us/tabs/team/team_repository.dart

```diff
@@ -15,77 +15,77 @@ abstract class TeamRepository {
   Future<List<TeamCategory>> getAllTeamMembers();
 }
 
-class FakeTeamRepository implements TeamRepository {
-  @override
-  Future<List<TeamCategory>> getAllTeamMembers() async {
-    // network delay
-    await Future.delayed(Duration(seconds: 1));
-    if (Random().nextBool()) {
-      throw NetworkException();
-    } else {
-      var response = {
-        "message": "Team members Fetched successfully.",
-        "data": [
-          {
-            "id": 1,
-            "name": "Kick Buttowski",
-            "profile_url":
-                "https://i.pinimg.com/originals/83/f5/88/83f588b8414b1bacfd0ef3027b5aba7f.jpg",
-            "image": null,
-            "member_type": "EXC",
-            "year": 2021,
-            "domain": "tech",
-            "linkedin": "http://linkdin.com/IamBttowski",
-            "facebook": "http://facebook.com/IamBttowski"
-          },
-          {
-            "id": 2,
-            "name": "Elon Musk",
-            "profile_url":
-                "https://upload.wikimedia.org/wikipedia/commons/8/85/Elon_Musk_Royal_Society_%28crop1%29.jpg",
-            "image": null,
-            "member_type": "HCD",
-            "year": 2021,
-            "domain": "tech",
-            "linkedin": null,
-            "facebook": null
-          }
-        ]
-      };
-
-      List<TeamCategory> categories = [
-        TeamCategory("Director", List()),
-        TeamCategory("Head of CDC", List()),
-        TeamCategory("Faculty Incharge", List()),
-        TeamCategory("Overall Co-ordinators", List()),
-        TeamCategory("Head Co-ordinators", List()),
-        TeamCategory("Managers", List()),
-        TeamCategory("Executives", List()),
-        TeamCategory("Other", List()),
-      ];
-
-      Map<String, int> typeToIndex = {
-        "DIR": 0,
-        "HCD": 1,
-        "FCT": 2,
-        "OCO": 3,
-        "HCO": 4,
-        "MNG": 5,
-        "EXC": 6,
-      };
-
-      (response["data"] as List).forEach((e) {
-        // converting json to dart TeamMember object.
-        TeamMember member = TeamMember.fromJson(e);
-        // adding the team member to the specific category.
-        // If that category doesnot exist in our directory, add it to the others list.
-        categories[typeToIndex[member.type] ?? 7].members.add(member);
-      });
-
-      return categories;
-    }
-  }
-}
+// class FakeTeamRepository implements TeamRepository {
```

Comment:
Any Specific Reasons for commenting entire FakeRepo?

Reply:
sir, we were trying to locate where fake team repository is implemented so that we can replace it with the actual api team repository. So we commented the fake repository.